crawler/.release_version

```diff
@@ -1 +1 @@
-1.8.1
```

Comment:
Why is there another `.release_version` file?

Reply:
Mistake :)